som_cas/templates/base.html

```diff
@@ -32,6 +32,7 @@
 
     {% block js %}
       <script src="{% static 'js/vendors/jquery-3.3.1.min.js' %}" ></script>
+      <script src="{% static 'js/vendors/popper.min.js' %}"></script>
```

Comment:
I see that finaly you have used bootstrap popovers, this import it's not necesary any more :)